### PR TITLE
Upgrading Electrum to v3.2.2

### LINKED ISSUE #81: Electrum fails to load transactions
Electrum v3.1.2 is failing to load transactions created by Electrum v3.2.2 on an online machine. Similar to #70, this makes it impossible to sign transactions within Bitkey cold-offline mode.

I've played around here and found out that the solution would be to update Electrum to the latest version v3.2.2 on BitKey. I'll attach a PR to this fix in a few moments.

### PR DESCRIPTION
**Pull request by vitork**
Upgrading Electrum to `v3.2.2` in order to fix issue with loading transactions from prior versions of Electrum.

Fixes #81

### CODE REVIEW

File: conf.d/04-wallets [hunk 1]
```diff
@@ -47,6 +47,14 @@ python setup.py install
...
 rm -r trezor-*
 
+# install electrum dependency python-typing
+tar -zxf typing-*.tar.gz
+rm typing-*.tar.gz
+mv typing-*/ $SHARE/python-typing/
+cd $SHARE/python-typing
+sudo python3 setup.py install
+cd $SRC
+
```
Comment (vitork) [inline conf.d/04-wallets:57]: `python-typing` is now bundled in Electrum v3.2.3, so we don't need to preinstall it anymore.

**Comment by estevaocm** [comment]
Hello, thank you for your request. My build workspace has been messed up with attempting to upgrade Bitkey do debian-stretch, and I haven't been able to fix it until tonight. I am attempting a build with Electrum 3.2.3 but the new Turnkey version is acting wierdly. Hopefully we'll accomplish a successfull build along the week.

**Comment by LF76** [comment]
Thanks, Estevão, for the great work! Hope soon we can have the updated ISO. Also with the latest available Electrum versions (DASH for example on Bitkey is still v.2.9.4)

**Comment by JedMeister** [comment]
Apologies BitKey is pretty much dead. I do have hopes to revive it at some point, but I just don't have enough spare cycles at the moment, so I'm not sure when that might be...

**Pull request closed without merging**